packages/net/hass/files/hass.conf

```diff
@@ -1,5 +1,6 @@
 config hass 'global'
-        option host 'ip.or.name.example:8123'
+        option host 'ip.or.name.example'
```

Comment:
why not just split/cut the previous host option to maintain backwards compatibility?

Reply:
Hmm, because now it is mandatory.
I could default to 80 if no port is given